api/src/api/libv2/api_xml.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# coding=utf-8
+# Copyright 2017 the Isard-vdi project authors:
+#      Josep Maria Viñolas Auquer
+#      Alberto Larraz Dalmases
+# License: AGPLv3
+import time
+from api import app
+from datetime import datetime, timedelta
+import pprint
+
+from rethinkdb import RethinkDB; r = RethinkDB()
+from rethinkdb.errors import ReqlTimeoutError
+
+import logging as log
+
+from .flask_rethink import RDB
+db = RDB(app)
+db.init_app(app)
+
+from ..libv2.isardViewer import isardViewer
+isardviewer = isardViewer()
+
+from .apiv2_exc import *
+
+from .ds import DS 
+ds = DS()
+
+from .helpers import _check, _parse_string, _parse_media_info, _disk_path
+
+class ApiXml():
+    def __init__(self):
+        None
+
+    def VirtInstallGet(self, id):
+        with app.app_context():
+            virt_install = r.table('virt_install').get(id).run(db.conn)
+        if virt_install == None:
```

Comment:
I don't get this. When using value comparisons I think it is better to use value comparison instead of reference comparison, as I got into some troubles before when comparing with 'is False'. I understand this:

-  == is for value equality. Use it when you would like to know if two objects have the same value.
-  is is for reference equality. Use it when you would like to know if two references refer to the same object.

(https://stackoverflow.com/questions/132988/is-there-a-difference-between-and-is)

Reply:
`is` compares if are the same object. None is a singleton so it is better use `is`. `False` isn't a singleton, it is an instance of `bool`.

Reply:
ok